integration-tests/test-api-manual.spec.js

```diff
@@ -63,7 +63,9 @@ describe('test-api-manual', () => {
       )
       assert.propertyVal(passedTest.content.meta, 'test.custom.tag', 'custom.value')
 
-      const customSpan = events.find(event => event.type === 'span')
+      const customSpan = events
+        .filter(event => event.content.type !== 'system' && event.content.name !== 'command_execution')
+        .find(event => event.type === 'span')
```

Comment:
no need to filter the whole array if you're only going to use the first matching element anyway no ?
```suggestion
        .find(event => event.content.type !== 'system' && event.content.name !== 'command_execution' && event.type === 'span')
```

Reply:
I did that way because it would be easier to add more exceptions just in case there were more cases like that. Since it's a test I prioritized clarity over performance. Anyway if it's too cumbersome for you feel free to commit the suggestion.

Reply:
Ok good idea, perfect :+1: